build.gradle

```diff
@@ -307,11 +307,13 @@ msgfmt {
 }
 
 test {
+    finalizedBy jacocoTestReport
```

Comment:
I don't think we want to directly create either a `dependsOn` or a `finalizedBy` relationship between the two tasks. This makes it so that no matter which task we run both will always run. I can see that a developer might want to only run unit tests locally, but not coverage, or, similarly, run coverage on an existing unit test report, without having to re-run the unit tests.

The best would probably be to remove both of these, and create a third task called `testAndCoverage`, that dependsOn test, and is finalyzedBy jacocoTestReport. And that task is called in the Jenkinsfile

Reply:
I think we should leave at least one direction of the relationship there. Tests can run without coverage, but running coverage implies running tests.

Reply:
ok, sounds reasonable. The 'run coverage on existing unit test report' use case is probably extremely rare if ever needed